### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by simplenotezy** [arash16/nuxt-ssr-cache#26]
From the looks of it this looks like a very interesting package - so good job so far!

I do, however, think this documentation needs some more information on what it caches exactly. Does it cache my axios / async data calls? How do I purge the cache? What are the expected load time difference?

**Comment by ChucKN0risK** [comment]
I agree with @simplenotezy . I would like to cache my static assets in production however I still don't understand if this package does what I'm looking for.

**Comment by ThunderDev1** [comment]
From what I can gather, it caches the nuxt.render output. This means the response time should be very fast as no api calls need to be made, and the rendering is already done. I don't think it will cache external assets. The cache key is route based, so adding a query param for example should enable you to fetch a 'fresh' version of the page.
